databuilder/models/user_elasticsearch_document.py

```diff
@@ -0,0 +1,62 @@
+import json
+from typing import List, Optional  # noqa: F401
+
+from databuilder.models.elasticsearch_document import ElasticsearchDocument
+
+
+class UserESDocument(ElasticsearchDocument):
+    """
+    Schema for the Search index document for user
+    """
+    def __init__(self,
+                 elasticsearch_index,  # type: str
+                 elasticsearch_type,   # type: str
```

Comment:
Do we need `elasticsearch_type`?

Reply:
Are you grouping the `User` and `Table` by different `elasticsearch_type`? If that is the case, could `elasticsearch_type` be a class variable in those classes instead of as a constructor parameter?

Reply:
yes, you could take a look at WIP people dag change(https://github.com/lyft/etl/pull/15621/files)

Reply:
@Alagappan , I would prefer do the refactor in a different pr.